onnx/onnx.in.proto

```diff
@@ -287,6 +287,12 @@ message GraphProto {
   // Initializers (see above) stored in sparse format.
   repeated SparseTensorProto sparse_initializer = 15;
 
+  // A list of named graphs, used to specify how model inputs should be 
+  // initialized. Each GraphProto must have a name (distinct from any other
+  // initializer) and one output, which is of the same type and shape as the
+  // corresponding model input.
```

Comment:
How about this?
> Initializers (see above) described by graphs. Each GraphProto must have a name (which acts as the initializer name), zero inputs, and one output (which acts as the initializer tensor).

Reply:
I think the type/shape constraint is implied by the spec already, but if you think it is worth being explicit, then IMO it should be on the `initializer` field since it applied to all initializers.

Reply:
I agree, this is why I didn't mention the type/shape constraint in the first iteration. The zero inputs requirement is also important. Thanks!